pybites_tools/alarm.py

```diff
@@ -10,28 +12,58 @@
 ALARM_MUSIC_FILE = os.environ["ALARM_MUSIC_FILE"]
 
 
-def countdown(seconds: int) -> None:
+def countdown_and_play_alarm(seconds: int, show_timer: bool = False) -> None:
     while seconds:
         mins, secs = divmod(seconds, 60)
-        print(f"{mins:02}:{secs:02}", end="\r")
+        if show_timer:
+            print(f"{mins:02}:{secs:02}", end="\r")
         time.sleep(1)
         seconds -= 1
 
-    print("00:00", end="\r")
+    if show_timer:
+        print("00:00", end="\r")
     playsound(ALARM_MUSIC_FILE)
 
 
-def main():
+def get_args():
     parser = argparse.ArgumentParser("Play an alarm after N minutes")
     parser.add_argument(
-        "-m", "--minutes", required=True, help="Number of minutes to countdown from"
+        "-m", "--minutes", required=True, help="Number of minutes to play alarm in"
     )
-    args = parser.parse_args()
-    seconds = int(args.minutes) * 60
-    try:
-        countdown(seconds)
-    except KeyboardInterrupt:
-        pass
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
+        "-b",
+        "--background",
+        action="store_true",
+        default=False,
+        help="Run timer in the background",
+    )
+    group.add_argument(
```

Comment:
I needed this extra arg otherwise in background mode, the timer would still persistently show